src/components/common/Input.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const Input = (props) => {
+  const {
+    type,
+    name,
+    value,
+    className,
+    placeholder,
+    handleChange,
+    icon,
+  } = props;
+  return (
+    <div className="field">
+      <div className="ui left icon input">
+        <input
+          type={type || "text"}
+          name={name}
+          className={className}
+          placeholder={placeholder || "Enter input"}
+          value={value}
+          onChange={handleChange}
```

Comment:
Make this required and default to an empty function in defaultProps like () => {}